modules/features/analytics-config/org.wso2.carbon.pc.analytics.config.feature/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <groupId>org.wso2.pc</groupId>
+        <artifactId>wso2pc-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../../..</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.carbon.pc.analytics.config.feature</artifactId>
+    <packaging>pom</packaging>
+    <name>WSO2 Carbon - Process Center - Analytics Configuration Feature</name>
+    <url>http://wso2.org</url>
+    <description>This feature contains the bundles required for Process Center Analytics Configurations</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.pc</groupId>
+            <artifactId>org.wso2.carbon.pc.analytics.config</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.wso2.maven</groupId>
+                <artifactId>carbon-p2-plugin</artifactId>
+                <version>${maven.carbon.p2plugin.version}</version>
```

Comment:
Thanks @Niranjan-K. This is already fixed in da118e5e29f041d612ef4a6a53402148b1074b42